src/Stripe.net/Services/Account/StripeAccountLegalEntityOptions.cs

```diff
@@ -7,228 +7,228 @@ public class StripeAccountLegalEntityOptions : INestedOptions
     {
         #region Address
 
-        [JsonProperty("legal_entity[address][city]")]
+        [JsonProperty("address[city]")]
```

Comment:
I don't fully grasp why this still works instead of requiring the `city` part to be in a sub-nested option

Reply:
The encoding will now automatically apply `legal_entity` from the `JsonProperty` attribute on the parent class ([here](https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Services/Account/StripeAccountSharedOptions.cs#L43)).

In order to maintain compatibility, I added some logic (in the `NewPrefix` method) to handle the case where the `JsonProperty` is `b[c]` in a nested class that's itself under a `JsonProperty` `a`. The resulting key is properly encoded as `a[b][c]`.

Of course, in most cases we should still move these attributes in dedicated classes, but we can do that in a followup PR.